nerdlets/pathpoint-nerdlet/components/Modal/TouchpointsEditor.js

```diff
@@ -557,7 +558,9 @@ class BodyTouchpointsEditor extends Component {
 
   HandleOnChange = (target, value, id) => {
     const { touchpoints, current } = this.state;
-    this.TestMeasureTime(current.touchpoint, value);
+    if (target === "queryMeasure") {
```

Comment:
This fixes a bug because this validation was being applied to EVERY text box in the form.